src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.HostProject.cs

```diff
@@ -60,6 +60,8 @@ public ProjectInfo CreateProjectInfoForCurrentState()
                     metadataReferences: _metadataReferences,
                     isSubmission: false,
                     hostObjectType: null);
+
+                return info.WithIsComplete(isComplete: false);
```

Comment:
@jasonmalinowski I thought you want it to be false? so, rather than we check it being misc workspace, we check this being incomplete and don't provide full semantic stuff.